dev/src/Command/SplitCommand.php

```diff
@@ -242,6 +249,7 @@ private function processComponent(
         $componentId = $component->getId();
         $isAlreadyTagged = $github->doesTagExist($repoName, $tagName);
         $defaultBranch = $github->getDefaultBranch($repoName) ?: 'main';
+        $branchToPush = $component->getBranch() ?? $defaultBranch;
```

Comment:
Let's just make `$branchToPush` an argument to the `processComponent` function. The `Component` class is about getting info about a component based on the codebase, so having a `branch` property on it doesn't really make sense.

Reply:
Done.